### DISCUSSION TITLE
Vercel Deployment Error

### DISCUSSION THREAD

**Discussion by Timo-Thees** [neuefische/questions#347] [Web] [answered]
Hello, 
Vercel tells me deployment has failed. This is the error message: 
![Bildschirmfoto 2022-11-30 um 15 52 21](https://user-images.githubusercontent.com/114575768/204830286-68c0c5c0-8f80-481a-8da2-878d8a146eb3.png)

 my understanding is that Vercel doesn't recognize the path for the component ''Editor'. In code the path looks like this:
![Bildschirmfoto 2022-11-30 um 15 53 00](https://user-images.githubusercontent.com/114575768/204831414-48a7d562-b4b7-43bf-999b-47140b5a5f26.png)

and the folder structure looks like this:
![Bildschirmfoto 2022-11-30 um 15 53 18](https://user-images.githubusercontent.com/114575768/204831597-7261df38-750a-44e7-9867-360d9f51a41f.png)

How can I resolve that?

**Comment by actyralf** [comment, marked as answer]
This is a very nasty error - the reason that Vercel cannot find your import is that git is not case-sensitive for file names by default. You renamed `editor.js` to `Editor.js` and adjusted the import accordingly. But git does not recognize the filename change at all, so it does not change (and upload) the filename. So on github the file is still called `editor.js` and the import from `Editor.js` will then fail.

Three steps to fix that:

1. Make git case-sensitive to prevent future occurences of this bug:
   Call `git config --global core.ignorecase false` in your terminal
2. Make sure that you are **in your project directory** and call `git config core.ignorecase false` to ensure that the setting is applied to your project's git configuration
3. Fix the existing problem by calling `git mv -f ./components/editor.js ./components/Editor.js`
   This should rename the file in your local repo to `Editor.js` correctly

**Reply by Timo-Thees** [reply]
Thank you! Solution 2 did work for me
